test/core/test_update.py

```diff
@@ -126,6 +139,174 @@ def test_update_global_attributes(self):
         self.assertEqual(13.5, ds2.attrs.get('geospatial_lat_max'))
         self.assertEqual(0.25, ds2.attrs.get('geospatial_lat_resolution'))
         self.assertEqual('degrees_north', ds2.attrs.get('geospatial_lat_units'))
-        self.assertEqual('2018-06-01T00:00:00.000000000', ds2.attrs.get('time_coverage_start'))
-        self.assertEqual('2018-06-05T23:00:59.000000000', ds2.attrs.get('time_coverage_end'))
+        self.assertEqual('2018-06-01T00:00:00.000000000',
+                         ds2.attrs.get('time_coverage_start'))
+        self.assertEqual('2018-06-05T23:00:59.000000000',
```

Comment:
`23:00:59` looks suspicious!

Reply:
I agree that it looks like a typo for `23:59:59`, but I think in this case it's just an arbitrarily chosen value.